tox.ini

```diff
@@ -52,7 +52,7 @@ description = Run the flake8 code quality checker.
 [testenv:mypy]
 deps = mypy
 skip_install = true
-commands = mypy --install-types --non-interactive --ignore-missing-imports sssom/ setup.py
+commands = mypy --install-types --non-interactive --ignore-missing-imports --implicit-optional sssom/ setup.py
```

Comment:
What does that do?

Reply:
In its absence `mypy` throws a bunch of errors.

Reply:
nothing you think is serious enough to be fixed though? I will trust you on this, just doing due diligence